auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -175,7 +163,6 @@ public void onSuccess(IdpResponse response) {
                 .signInWithCredential(credential)
                 .addOnCompleteListener(new CredentialSignInHandler(
                         this,
-                        getSaveSmartLock(),
```

Comment:
Here's where it works because we initialize the fragment before Auth can give us a callback.